README.md

```diff
@@ -897,6 +898,29 @@ const scope = nock('http://api.myservice.com')
   })
 ```
 
+### Axios
+
+As axios is very likely to use in many js apps, we would like to show the case by following sample code.
+For more detail you can go to https://github.com/nock/nock/issues/699#issuecomment-272708264
+```js
+// my-remote-call.test.js
+import test from 'ava';
+import axios from 'axios';
+import httpAdapter from 'axios/lib/adapters/http';
+import nock from 'nock';
+
+const host = 'http://localhost';
+axios.defaults.host = host;
+axios.defaults.adapter = httpAdapter;
+
+test('remote call with axios should pass', async t => {
+  const call = nock(host)
+    .get('/test').reply(200, 'test data');
+  await axios.get('/test');
+  t.is(call.isDone(), true);
+});
+```
```

Comment:
Hi! Thanks for opening this.

Could I suggest using this snippet (assuming it works)? It matches the naming and style of the rest of nock + the readme, doesn't set host which isn't strictly necessary, and adds some comments.

```js
import axios from 'axios'
import nock from 'nock'
import test from 'ava' // You can use any test framework.

// If you are using jsdom, axios will default to using the XHR adapter which
// can't be intercepted by nock. So, configure axios to use the node adapter
// instead.
//
// References:
// https://github.com/nock/nock/issues/699#issuecomment-272708264
// https://github.com/axios/axios/issues/305
axios.defaults.adapter = require('axios/lib/adapters/http')

test('can fetch test response', async t => {
  // Set up the mock request.
  const scope = nock('http://localhost')
    .get('/test')
    .reply(200, 'test response')

  // Make the request. Note that the hostname must match exactly what is passed
  // to `nock()`. Alternatively you can set `axios.defaults.host = 'http://localhost'`
  // and run `axios.get('/test')`.
  await axios.get('http://localhost/test')

  // Assert that the expected request was made.
  scope.done()
})
```

Also, there's a section "Common issues" in the readme. Would you mind moving this new section there?

I hadn't read those threads before – it's good to know about!

Reply:
I like u idea. I modify my change or u modify my change?

Reply:
I like u idea. I modify my change or u modify my change?

Reply:
I updated yours!